centos8/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM centos:8
 LABEL maintainer="Shaun Jackman <sjackman@gmail.com>"
 LABEL name="linuxbrew/centos8"
 
-RUN yum install -y curl file make ruby sudo which \
+RUN yum install -y git curl file make ruby sudo which glibc-locale-source glibc-langpack-en \
```

Comment:
Please sort these packages in alphabetical order.

Reply:
Ok, done.